src/Region/Ds9ImportExport.cc

```diff
@@ -503,34 +497,24 @@ RegionState Ds9ImportExport::ImportEllipseRegion(std::vector<std::string>& param
         // Control points in pixel coordinates
         std::vector<CARTA::Point> control_points;
         if (_pixel_coord) {
-            CARTA::Point point;
-            point.set_x(param_quantities[0].getValue());
-            point.set_y(param_quantities[1].getValue());
-            control_points.push_back(point);
-            point.set_x(param_quantities[2].getValue());
-            point.set_y(param_quantities[3].getValue());
-            control_points.push_back(point);
+            control_points.push_back(Message::Point(param_quantities[0].getValue(), param_quantities[1].getValue()));
+            control_points.push_back(Message::Point(param_quantities[2].getValue(), param_quantities[3].getValue()));
```

Comment:
Because we sometimes use different indices, the utility functions would need optional parameters -- then this could be written as
```
control_points.push_back(Message::Point(param_quantities));
control_points.push_back(Message::Point(param_quantities, 2, 3));
```